addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -610,6 +610,68 @@ QUnit.test('chat window: open / close', async function (assert) {
     );
 });
 
+QUnit.test('chat window (mobile): open / close with no server notification', async function (assert) {
```

Comment:
Would be best if there was one test for open and one test for close. (the flux is the same but split the assertions)

Reply:
Done